packages/admin-panel/package.json

```diff
@@ -32,6 +32,7 @@
     "@material-ui/lab": "^4.0.0-alpha.47",
     "@material-ui/styles": "^4.9.10",
     "@tupaia/access-policy": "workspace:*",
+    "@tupaia/tsutils": "workspace:*",
```

Comment:
Given that this is a frontend package and we had that issue with `tsutils` not building well on the frontend in Tupaia, will this cause any issues?

Reply:
Good thinking! I’ll test this on a remote deployment today

Reply:
You were right about this; same issue we saw on DataTrak happens when the VizBuilder is loaded

Fixed, and slightly improved logic robustness along the way. Thank you!